srvclient.go

```diff
@@ -113,6 +128,45 @@ func SRVNoPort(hostname string) (string, error) {
 	return addr[:strings.Index(addr, ":")], nil
 }
 
+// AllSRV returns the list of all hostnames and ports for the SRV lookup
+// The results are sorted by priority and then weight.
```

Comment:
Sorry, one last thing, we should document that this has the same port behavior as the `SRV` method, but it replaces the port on all results.